src/components/Problems/Problems.tsx

```diff
@@ -56,6 +50,14 @@ export const Problems: FC = () => {
     queryFn: () => axios.get<SeriesWithProblems>(`/api/competition/series/${id.seriesId}`),
     enabled: id.seriesId !== -1,
   })
+
+  const {data: bannerMessage, isLoading: isBannerLoading} = useQuery({
+    queryKey: [id.seriesId],
```

Comment:
queryKey by mal matchovat API route segmenty - nemame to nijak automatizovane, no je to globalny kluc react-query, pod ktorym cachuje response, tak cely kluc by mal odrazat, o ake data ide. tu teda asi `['cms', 'info-banner', id.seriesId]`